CMakeLists.txt

```diff
@@ -161,10 +161,25 @@ ELSE()
   ENDIF()
   ADD_DEBUG_COMPILE_FLAG("-O0")
   ADD_DEBUG_COMPILE_FLAG("-g3")
-  ADD_NONDEBUG_COMPILE_FLAG("-O2")
+  IF (EMSCRIPTEN)
+    # really focus on minimizing output size when compiling sources
+    ADD_NONDEBUG_COMPILE_FLAG("-Oz")
```

Comment:
How much of a performance hit is this? I wonder how much we need to be minimizing size for offline node usage. It might be good to use -Oz only with MinSizeRel builds.

Reply:
It is about half the size (!) and I didn't notice a big difference in speed. But I didn't check carefully, as this PR just keeps the same flags as before, it just moves to cmake.